todoapp/users/serializers.py

```diff
@@ -1,10 +1,59 @@
 from django.contrib.auth import authenticate
 from django.contrib.auth import get_user_model
 from django.contrib.auth.hashers import make_password
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
 
 from rest_framework import serializers
 from rest_framework.authtoken.models import Token
+from users.models import CustomUser
 
 
-# Add your serializers
+class CustomUserSerializer(serializers.ModelSerializer):
+    def __init__(self, *args, **kwargs):
+        exclude = kwargs.pop("exclude", None)
+        super().__init__(*args, **kwargs)
+
+        if exclude is not None:
+
+            not_allowed = set(exclude)
+            existing = set(self.fields)
+
+            for field_name in not_allowed & existing:
+                self.fields.pop(field_name)
+
+    class Meta:
+        model = CustomUser
+        fields = ["id", "first_name", "last_name", "email"]
+
+
+class UserTodosStatsSerializer(CustomUserSerializer):
+    completed_count = serializers.IntegerField()
+    pending_count = serializers.IntegerField()
+
+    class Meta:
+        model = CustomUser
+        fields = [
+            "id",
+            "first_name",
+            "last_name",
+            "email",
+            "completed_count",
+            "pending_count",
+        ]
```

Comment:
We can make use of parent's Meta class
```suggestion
class Meta(CustomUserSerializer.Meta):
   fields = CustomUserSerializer.Meta.fields + ["completed_count", "pending_count"]
```